en/app.ftl

```diff
@@ -300,3 +300,17 @@ survey-option-not-likely = Not likely
 vpn-promo-headline = Save 50% with a full year subscription
 vpn-promo-copy = Protect your online data and choose a VPN subscription plan that works for you.
 vpn-promo-cta = Get { -brand-name-mozilla-vpn }
+
+## Email wrapping (header and footer messages wrapped around forwarded emails)
+
+# Variables:
+#   $display_email (string) - This is the relay email address. Example: abc123@relay.firefox.com
```

Comment:
Is Relay or relay, and should it  be treated as a product name?

Reply:
Updated this doc.